src/CommunityToolkit.Maui/Views/RatingView/RatingView.shared.cs

```diff
@@ -0,0 +1,627 @@
+﻿// Ignore Spelling: color, bindable, colors
+
+using System.Windows.Input;
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <summary>Rating view control.</summary>
+public class RatingView : TemplatedView, IRatingView
```

Comment:
will we support negative values for RatingView?

Reply:
This is an excellent point, and I never thought about it.  I'll put in a Property validator and also a Clamp to make sure.